### ISSUE TITLE
'Value and impact' is not really an activity

### ISSUE THREAD

**Issue by bvhme** [publiccodenet/about#658]
@Ainali mentioned that 'Value and Impact' is not really an activity.

What is this and where should it go? I'm a bit at a loss, so I don't have an immediate proposed solution.

**Comment by lscheske** [comment]
I agree that "Value & Impact" are not activities and may be best located somewhere else. They are more on the level of mission statements and justify why we exist. I see two possible locations: 

* A lot of this content should (and will) be reflected on the website (publiccode.net)
* It would be well placed on the About landing page, as a new point under "Information & Processes"

**Comment by ElenaFdR** [comment]
'Evaluating our value and impact' is an activity we will need at some point.

**Comment by bvhme** [comment]
Definetely, it is however not really aligned with the current content of that 'activity' folder as I understand it which is more 'mapping of value and impact'.

They could be under `/organization/value-propositions/` or something.

**Comment by clausmullie** [comment]
Personally I don't think the pages under the `value` heading fitinto `/organization/`, as this is not about our organisation, how we are structured/organised, or something we define/decided about ourselves. It is more of a mapping/tracking/measuring/research activity that we do about how other people perceive the the value of our work.

My sense is that the `all press` page should be moved from `About/` to `publiccode.net/`, as per issue [#52](https://github.com/publiccodenet/publiccode.net/issues/52), and the policy documents likewise, as per issue [#205](https://github.com/publiccodenet/publiccode.net/issues/205). 

The sustainable development goals could then either move to `/organization/` if they are a conscientious goal of ours, or `publiccode.net/` if this page is a marketing asset.

**Comment by bvhme** [comment]
Great observations @clausmullie. I think you're right on this.

I think a part of the value proposition is probably policy, so can live under something like `/organization/` whilst part of it is organizational development (which might be an activitiy) and part is storytelling (which is more for <publiccode.net>)

In addition, I think it is totally fine for things like conscientious goals of ours to live in both `/organization/`, where we describe it as policy for our own organization and on publiccode.net where we try to explain it to the world. I think you most often see this in organizations, albeit usually not both open.

**Comment by clausmullie** [comment]
Jan and I would like to propose the following fix for this issue.

Our thinking is that: 

- anything which shows proven impact/mention should be on the marketing site
- anything which is an activity can go straight into the activities index
- the value/impact umbrella is no longer needed

Actions to be taken inlcude:

- [ ] Content from press.md should move to publiccode.net
- [ ] Policy documents that mention and refer to the Foundation for Public Code should move to publiccode.net
- [ ] sustainable-development-goals.md should move to the bullet point list "This is how we approach our work:" in /organization/
- [ ] Turn /user-mapping/ into own activitiy on about/activities/
- [ ] Turn /pain-points/ into own activity on about/activities/ (perhaps changing name to Understanding our users)
- [ ] Remove value-impact page 
- [ ] Care should be taken to ensure all redirects are in place

@ericherman and @bvhme what are your thoughts on this?

**Comment by ericherman** [comment]
> Jan and I would like to propose the following fix for this issue.

This is a very good plan.

**Comment by Ainali** [comment]
> @Ainali @clausmullie is enough done here to close this issue?

I still think we should do the identified things that are unchecked. This issue has only been deprioritized due to other more urgent matters.

**Comment by clausmullie** [comment]
This work is dependent on getting the publiccode.net revamp work done first, so that we have a place where we can move things to (ie the press page)
